app/javascript/packs/bus-stop.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import capitalize from './utils/capitalize'
+
+const BusStop = props => {
+  const { predictions, routeName } = props.busStop
+  return (
+    <div>
+      <h1>
+        Hey there, {routeName} bus stop
+      </h1>
+      {
+        Object.keys(predictions).map((predictionKey) => {
+          return (
+            <p key={routeName+"-"+predictionKey}>
+              {capitalize(predictionKey)}: {predictions[predictionKey]}
```

Comment:
Does this pull the underscores out of the keys?

Reply:
nope.